packages/css/src/types.ts

```diff
@@ -436,11 +431,15 @@ export interface ThemeUIExtendedCSSProperties
     AliasesCSSProperties,
     OverwriteCSSProperties {}
 
+export type StylePropertyValue<T> =
```

Comment:
Any property in a style object is either ResponsiveStyleValue<T> (which includes T), a function which returns such value or a nested style object.